storybook/src/stories/Forms/VpeForm.stories.js

```diff
@@ -115,7 +115,20 @@ const VpeFormStory = () => {
         vpc: "workload",
         service: "kms",
         resource_group: "test",
-        security_groups: ["1", "2", "3"],
+        security_groups: [
+          {
+            name: "1",
+            vpc: "workload",
+          },
+          {
+            name: "2",
+            vpc: "workload",
+          },
+          {
+            name: "3",
+            vpc: "management",
+          },
+        ],
```

Comment:
`security_groups` should be list like `["1"]` and `securityGroups` should contain this object

Reply:
currently cant select any sg

Reply:
whoops